frontends/election-manager/src/screens/reports_screen.tsx

```diff
@@ -183,6 +184,11 @@ export function ReportsScreen(): JSX.Element {
     </p>
   );
 
+  const canSaveResults =
+    !cvrFilesQuery.isLoading &&
+    !!cvrFilesQuery.data &&
```

Comment:
One more place to handle error explicitly. Could just say `cvrFilesQuery.isSuccess && cvrFilesQuery.data.length > 0`